dm/config/task.go

```diff
@@ -278,6 +281,14 @@ func NewTaskConfig() *TaskConfig {
 	return cfg
 }
 
+func (c *TaskConfig) verifyTaskName() bool {
+	if c.Name == "" {
+		return false
+	}
+
+	return taskNameRegexp(c.Name)
```

Comment:
return `taskNameRegexp(c.Name)` is enough, no need to check `c.Name == ""`?

Reply:
[Done](https://github.com/pingcap/dm/pull/290/files#diff-b263153ff75f30e0c0f3fd51160160faR294)